central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -72,7 +75,16 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 
 	// if not ready, add it to the status errors
 	if readyPods < desiredPods {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready.  Only %d pods are ready when %d are desired.", readyPods, desiredPods))
+		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready. Only %d pods are ready when %d are desired.", readyPods, desiredPods))
+	}
+
+	// we support only newer versions of compliance operator
+	minVersion, _ := semver.NewVersion(minimalComplianceOperatorVersion)
+	complianceOperatorVersion, err := semver.NewVersion(operatorInfo.GetVersion())
+	if complianceOperatorVersion == nil || err != nil {
+		operatorErrors = append(operatorErrors, fmt.Sprintf("invalid compliance operator version %q", operatorInfo.GetVersion()))
+	} else if complianceOperatorVersion.LessThan(minVersion) {
+		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator version %q is not supported. Minimal required version is %q", operatorInfo.GetVersion(), minimalComplianceOperatorVersion))
```

Comment:
Please check the capitalization rules. We have 2 sentences here but only one starts with capital and end with period respectively.